### DISCUSSION TITLE
[Small fix] [Search] [UI Accuracy] Make search panel bigger

### DISCUSSION THREAD

**Discussion by CorruptionVR** [DustinBrett/daedalOS#383] [Ideas]
hey, this is a small detail, but I noticed that the search panel in actual Windows is taller and wider than it is in daedalOS.
this is daedalOS
![daedalOS](https://github.com/DustinBrett/daedalOS/assets/78228404/3d4504b2-1442-4955-934a-6471677eb27a)
this is Windows
![Windows](https://github.com/DustinBrett/daedalOS/assets/78228404/887a2fad-7aed-44d7-a512-f90ead9562ea)
both screenshots were taken on the same computer and I enabled small taskbar icons

edit: I also just noticed that in Windows, when you press the start key, and then start typing, it opens search, but this functionality is not present in daedalOS

**Comment by DustinBrett** [comment]
Thanks for the mention. Indeed I took some liberties with the search, which will evolve over time as I only just made it. But at the moment that height is my preference as it fits better with the content and in small viewports.

As for typing opening search, it actually is present, so if that is not working it's a bug. And indeed I see it's not working, so I will see what is up there, cause it did work earlier, but I haven't yet made all tests for the search on the taskbar.

**Reply by DustinBrett** [reply]
I have fixed the search issue, will publish later tonight likely.

https://github.com/DustinBrett/daedalOS/commit/c879815c3626bf797a673260d836b7d3cc424755

**Comment by Pixel556** [comment]
It's actually based on the size of your start menu. Therefore, it is already fixed
